src/UI/UI.php

```diff
@@ -6,6 +6,14 @@
 
 class UI
 {
+    public function editable($model, $field)
+    {
+        return view('laravel-views::components.editable', [
```

Comment:
You could use `$this->component` instead, like the `attributes` method, you'll see it once yo update it with master